### PR TITLE
Adds in a Sports car.

### PR DESCRIPTION
**Pull request by Nienhaus**
Sprites by Walter, changed to have a roof.
u1f5q5k

### CODE REVIEW

File: code/modules/urist/gamemodes/scom/sciencedatums.dm [hunk 1]
```diff
@@ -113,6 +113,12 @@
 	hidden = 1
 	resources = 800
 
+/datum/scomscience/recipe/sportscar
```
Comment (Glloyd) [inline code/modules/urist/gamemodes/scom/sciencedatums.dm]: take this out for now, I'm tweaking the science system/progression a bit at the moment.

**Comment by Glloyd** [comment]
only real issue with this, spawn some hostile mobs in, drive around in one and let me know what happens. I'm not sure what will happen with mobs and 64x64 vehicles. I feel like the vehicle itself won't be destroyed, and the person riding it will be stuck underneath it if they get knocked out.

**Comment by TGameCo** [comment]
snazzy

File: code/modules/vehicles/vehicle.dm [hunk 2]
```diff
@@ -297,6 +299,9 @@
 			C.pixel_y += load_offset_y
 		C.layer = layer + 0.1		//so it sits above the vehicle
 
+	if(mob_offset_x)
+		C.pixel_x += mob_offset_x
```
Comment (scrdest) [inline code/modules/vehicles/vehicle.dm]: This will cause the mob to be offset by (load_offset_x+mob_offset_x)

**Comment by scrdest** [comment]
@Glloyd I tested it, trying to get it to the snazzier original roofless sprite; the alien empress shot the car to pieces, it exploded and left me knocked on the floor. A syndicate swordsman attacked me, not the car, and I assume your issue is that you cannot get the wounded out of the car.

That could be remedied by making it roofless again, or having the person be encased inside, a'la mechs.

**Comment by scrdest** [comment]
Figured out how to make the roofless version work properly.

**Comment by Glloyd** [comment]
Yeah, my concern was the hostile mob hitting the mob riding it, and them getting stuck behind the layer.

**Comment by scrdest** [comment]
Nienhaus pointed out that clicking on the car ejects the rider. Anyhow, it works as a roofless one now.

**Comment by Nienhaus** [comment]
Just like the cargo train, click it and it forces them out. Also, updated to roofless.

**Comment by scrdest** [comment]
It basically IS a cargo train until someone moves all urist vehicles to their own code.

**Comment by Nienhaus** [comment]
@Glloyd Is this fine yet?

**Comment by Glloyd** [comment]
:+1:

**Pull request merged**